pandas/io/formats/format.py

```diff
@@ -625,7 +625,7 @@ def to_string(self):
                 max_len += size_tr_col  # Need to make space for largest row
                 # plus truncate dot col
                 dif = max_len - self.w
-                adj_dif = dif
+                adj_dif = dif + 1
```

Comment:
do you have any clue why this was needed? (I mean: a clue enough to add a comment?)

Reply:
Also wondering: would it be possible to somehow test this? (but then we would need to make clear to this code the terminal has a certain size ...)

Reply:
*Ahem*, like I said, I *think* that this might have to do with a space separator that isn't needed for the end of the last column. But I'm not sure, and it is a lot of work to debug and keep track of all values to hunt down the true culprit...

Reply:
maybe then put a comment just referring to this PR